README.md

```diff
@@ -106,6 +106,52 @@ fn method_name() {
 }
 ```
 
+##Full Example
+```rust
+use std::fs::File;
+
+use flame;
+#[macro_use] extern crate flamer;
```

Comment:
I think `#[macro_use] extern crate _` is no longer needed on Rust 2018.

Either we remove it or add a comment that it's only needed on the older edition.

Reply:
Yup you're right and since we aren't using the pre 2018 import stuff it wouldn't make much sense to keep it, I will fix it up and make a PR to `flame`

Reply:
I triple checked you do in fact still need `#[macro_use] extern crate _` I'm include a [playground](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=a583ab492771f52030f37232a31fd8d1) only because I was surprised too, I really thought that was changed for macros.

Reply:
You need to [`use clap::clap_app` (playground)](https://play.rust-lang.org/?version=nightly&mode=debug&edition=2018&gist=968e4c0df656113600b73ca2a2dff92e) if you don't `#[macro_use]` which imports all macros unconditionally.

Reply:
Ah I see sorry, thanks for explaining that.